### PR TITLE
Firebase

### PR DESCRIPTION
**Pull request by Szpadzik**
at the moment I've connected React with Firebase, nowk tasks for me: 
FIREBASE'S AUTHENTICATION API & SIGN UP WITH REACT AND FIREBASE

### CODE REVIEW

**Comment by michalczukm** [comment]
Waiting for your changes, @Szpadzik please ping me when you push them ;)

File: .env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Firebase
```
Comment (michalczukm) [inline .env:1]: hmm we shouldn't have this file on repo

**Review by michalczukm — CHANGES_REQUESTED** [review]
I know that this PR is WIP.

While working on it, please remove all `console.log` leftovers, and change all `Test` component names to more meaningful :)

File: .env (hunk 1 above)
Comment (Szpadzik) [reply .env:1]: but i added this one to gintignore ?

Comment (michalczukm) [reply .env:1]: you've added directory, not file 
Is `.env/`, should be `.env`

File: src/components/MarketDetail.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import firebase from '../components/Firebase/FirebaseConfig';
+import {useParams} from 'react-router-dom';
+import photo from "../assets/Webp.net-resizeimage.png";
+
+
+function useData(id) {
+    const [data, setData] = useState(null);
+
+    useEffect(() => {
+        firebase.firestore().collection('warszawa').doc(id)
```
Comment (michalczukm) [inline src/components/MarketDetail.js:11]: looks like we should pass here also `city` - right now you have hardcoded "warszawa"

File: .env (hunk 1 above)
Comment (Szpadzik) [reply .env:1]: hmmmm before i added only .env and was the same situation, in this moment i added .env

**Comment by Szpadzik** [comment]
@michalczukm i added readme, but here on firebase branch:/ could you look on this, is correct, maybe i should added more info?

**Review by michalczukm — APPROVED** [review]

**Pull request merged**